scripts/setup/uninstall.ps1

```diff
@@ -121,7 +125,10 @@ function Uninstall($_path)
             }
         }
     }
+
+    Remove-LocalGroup -Name $IISAdminOwnersGroup -ErrorAction "Continue"
```

Comment:
We can't just delete that group in uninstall. We don't know if we created it. If we knew we created it then we could.

Reply:
I will assign a description `This group is used by IIS Administration API and members have full access to the application. It would be removed during if the application is uninstalled` and verify if the description is there before removing